src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/blob.py

```diff
@@ -0,0 +1,144 @@
+# ---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# ---------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+from datetime import datetime
+from azure.cli.command_modules.storage._command_type import cli_storage_data_plane_command
+from azure.cli.command_modules.storage._factory import blob_data_service_factory
+from azure.storage.blob import BlockBlobService
+from azure.storage.blob.models import Include
+
+
+def storage_blob_copy(client, source_account, source_container, destination_container,
+                      source_sas=None,
+                      pattern=None, recursive=False, snapshots=False,
+                      exclude_old=False, exclude_new=False, blob_type="block"):
+    """
+    Copy blobs between containers and storage accounts. This is a server-side copy operation
+    therefore the command is asynchronous.
```

Comment:
What parameters does AzCopy support? Curious which ones are 1) required by the API copy operation 2) ported from AzCopy and 3) newly added for Az.